app/scripts/modules/core/help/helpField.directive.js

```diff
@@ -50,6 +51,22 @@ module.exports = angular
           scope.$watch('key', applyContents);
           scope.$watch('fallback', applyContents);
           scope.$watch('content', applyContents);
+
+          let tooltipShownStart = null;
```

Comment:
it would have been nice if we could have just used the `popover-popup-delay` property but we have no triggers in UIBS to let the user know when it's been opened or closed.

i've reached out to another dev asking about whether or not this might make sense - adding an `afterOpen` and `afterClose` event handlers.